compiler/rustc_feature/src/removed.rs

```diff
@@ -60,7 +60,7 @@ declare_features! (
     (removed, const_compare_raw_pointers, "1.46.0", Some(53020), None,
      Some("cannot be allowed in const eval in any meaningful way")),
     /// Allows non-trivial generic constants which have to be manually propagated upwards.
-     (removed, const_evaluatable_checked, "1.48.0", Some(76560), None, Some("renamed to `generic_const_exprs`")),
+     (removed, const_evaluable_checked, "1.48.0", Some(76560), None, Some("renamed to `generic_const_exprs`")),
```

Comment:
It's possible this token is tied to an older version of rust and thus not changeable. Dunno...

Reply:
this should indeed not be changed